dotcom-rendering/src/components/FrontSectionTitle.tsx

```diff
@@ -150,7 +132,7 @@ export const FrontSectionTitle = ({ title, collectionBranding }: Props) => {
 			return <div css={titleStyle}>{title}</div>;
 		}
 		default: {
-			assertUnreachable(collectionBranding);
+			assertUnreachable();
```

Comment:
This should not change
```suggestion
			assertUnreachable(collectionBranding);
```

Reply:
but that value isn't being used? not sure I understand :(

Reply:
The point of the `asserUnreachable` method is to assert that the code is unreachable, as explained [in the adjoining article](https://tinytip.co/tips/ts-switch-assert-unreachable/).

The reason why TypeScript is now complaining is because we’ve changed the underlying type from a discriminated union to a single type, so it’s no longer narrowed effectively through the switch case.

Here’s a branch which reinstates the existing behaviour: https://github.com/guardian/dotcom-rendering/compare/remove-editorial-badges...mxdvl/fix-assert-unreachable